exercises/largest-series-product/instructions.md

```diff
@@ -0,0 +1,59 @@
+# Instructions
+
+Your task is to look for patterns in the long sequence of digits in the encrypted signal.
+
+The technique you're going to use here is called the largest series product.
+
+Let's define a few terms, first.
+
+- **input** - the long sequence of digits that you need to analyze
+- **series** - a sequence of adjacent digits that is contained within the input
+- **span** - how many digits long each series is
+
+To form a series, take digits that are next to each other in the original input.
+
+Given the input `"1234"`, if you are working with a span of `3`, there will be two possible series:
+
+- `"123"`
+- `"234"`
+
+The **product** is what you get when you multiply numbers together.
+The product of `2` and `3` is `6` (`2 × 3 = 6`).
+
+So the product of a series is the result of multiplying all the digits in a series together.
+
+- The product of the series `"123"` is 6 (`1 × 2 × 3 = 6`).
+- The product of the series `"234"` is 24 (`2 × 3 × 4 = 24`).
+
+For a series of zero digits, you need to return the empty product (the result of multiplying no numbers), which is 1.
```

Comment:
```suggestion
Note: For a series of zero digits, you need to return the empty product (the result of multiplying no numbers), which is 1.
```

I do wonder if we could just remove the test case for the series of zero digits from the exercise to simplify this whole bit.

Reply:
Yeah, that would actually be better. We don't have a way of doing this with configlet though.